schema/musicxml.xsd

```diff
@@ -5320,6 +5316,17 @@ If the parentheses attribute is set to yes, the notehead is parenthesized. It is
 		</xs:simpleContent>
 	</xs:complexType>
 
+	<xs:complexType name="release">
+		<xs:annotation>
+			<xs:documentation>The release type indicates that a bend is a release rather than a normal bend or pre-bend. The offset attribute specifies where the release starts in terms of divisions. The first-beat and last-beat attributes of the parent bend element are relative to the original note position, not this offset value.</xs:documentation>

```

Comment:
Is `offset` relative to the onset of this `<note>`, or relative to the global position? I think this is worth clarifying.